Files/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -45,7 +50,7 @@
         </ResourceDictionary>
     </ContentDialog.Resources>
 
-    <Border MaxWidth="500">
+    <Border Width="500">
```

Comment:
Is it possible to use a smaller width when the user is just deleting items? 500 is a little wide for that scenario.